### PR TITLE
chrisjurich/loop_refinement_tutorial: PR for HTP dev subgroup

### PR DESCRIPTION
**Pull request by chrisjurich**

### CODE REVIEW

File: docs/source/sci_api_tutorial/add_missing_residues.rst [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+
+
+Briefs
+============================================
+
+Proteins are often missing residues from flexible loop regions. This is a problem for various types
+of calculations including molecular dynamics (MD) and quantum mechanics (QM) simulations. This function
+aids in HTP workflow design by adding missing ``Residue()`` objects into a ``Structure()``. Missing residues
+can be defined by the user or by pulling from the Protein DataBank (PDB) when possible.
```
Comment (shaoqx) [inline docs/source/sci_api_tutorial/add_missing_residues.rst:13]: Good job! I like how you describe this question

File: docs/source/sci_api_tutorial/add_missing_residues.rst [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+**You will find a detailed tutorial of how to use this Science API.**
+
+Input/Output
+=========================================================================================
+
+.. panels::
+
+    :column: col-lg-12 col-md-12 col-sm-12 col-xs-12 p-2 text-left
+
+    .. image:: ../../figures/add_missing_residues.svg
```
Comment (shaoqx) [inline docs/source/sci_api_tutorial/add_missing_residues.rst:39]: Please adjust the layout to make font larger

File: docs/source/sci_api_tutorial/add_missing_residues.rst [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+        :width: 100%
+        :alt: dipole_bond
+
+Input
+-----------------------------------------------------------------------------------------
+.. panels::
+
+    :column: col-lg-12 col-md-12 col-sm-12 col-xs-12 p-2 text-left
+
+    The parameters of the ``add_missing_residues`` API contain the following options
```
Comment (shaoqx) [inline docs/source/sci_api_tutorial/add_missing_residues.rst]: This can be removed

File: docs/source/sci_api_tutorial/add_missing_residues.rst [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+
+
+**Input**: The ``stru``, ``missing_residues``, ``method`` arguments and required inputs
+
+    - ``stru``:
+        A ``Structure()`` object that is missing residues. 
+
+        .. admonition:: Getting a ``Structure``
+
+            | A ``Structure()`` object can be obtained by using various parsers including the ``PDBParser()`` from ``structure_io`` submodule.
```
Comment (shaoqx) [inline docs/source/sci_api_tutorial/add_missing_residues.rst]: Please also link it to the how-to-obtain-xxx page (see QZ's branch)

File: docs/source/sci_api_tutorial/add_missing_residues.rst [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+
+    - ``missing_residues``:
+        A ``List[SeqRes]`` objects which describe the chain/index location and identity of each missing residue.
+
+        .. admonition:: Getting the missing residues
+
+            | A ``List[SeqRes]`` objects can be created by giving the four letter PDB code of a structure using the ``identify_missing_residues()`` function from the ``preparation`` module.
```
Comment (shaoqx) [inline docs/source/sci_api_tutorial/add_missing_residues.rst]: Please provide a link to the required API (identify_missing_residues)

File: docs/source/sci_api_tutorial/add_missing_residues.rst [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+
+
+Output
+-----------------------------------------------------------------------------------------
+.. panels::
+
+    :column: col-lg-12 col-md-12 col-sm-12 col-xs-12 p-2 text-left
+
+    **Output**: A ``Union[None, Structure]`` that is either nothing or a copied ``Structure`` with the added missing residues.
+
```
Comment (shaoqx) [inline docs/source/sci_api_tutorial/add_missing_residues.rst]: Please remove the redundant formt (e.g.: panel, `**Output**`, etc.)

**Review by shaoqx — CHANGES_REQUESTED** [review]
Very nice! I like everything about in this page of the tutorial.
Just some modifications needed.
Please see the comments.

File: docs/source/sci_api_tutorial/add_missing_residues.rst [hunk 7]
```diff
@@ -0,0 +1,138 @@
...
+            A(polypeptide): residue: 4-254,256-300 atom_count: 2339
+            B(polypeptide): residue: 4-252,259-300 atom_count: 2310
+        )
+        
+        <Structure object at 0x7f8576baa370>
+        Structure(
+        chains: (sorted, original ['A', 'B'])
+            A(polypeptide): residue: -1-304 atom_count: 2408
+            B(polypeptide): residue: -1-304 atom_count: 2408
+        )
```
Comment (shaoqx) [inline docs/source/sci_api_tutorial/add_missing_residues.rst:130]: Add author info (reference develop branch)

File: docs/source/sci_api_tutorial/add_missing_residues.rst (hunk 3 above)
Comment (chrisjurich) [reply docs/source/sci_api_tutorial/add_missing_residues.rst]: Removed!

File: docs/source/sci_api_tutorial/add_missing_residues.rst (hunk 4 above)
Comment (chrisjurich) [reply docs/source/sci_api_tutorial/add_missing_residues.rst]: Done!

File: docs/source/sci_api_tutorial/add_missing_residues.rst (hunk 7 above)
Comment (chrisjurich) [reply docs/source/sci_api_tutorial/add_missing_residues.rst:130]: Added!

File: docs/source/sci_api_tutorial/add_missing_residues.rst (hunk 6 above)
Comment (chrisjurich) [reply docs/source/sci_api_tutorial/add_missing_residues.rst]: Got it!

File: docs/source/sci_api_tutorial/add_missing_residues.rst (hunk 2 above)
Comment (chrisjurich) [reply docs/source/sci_api_tutorial/add_missing_residues.rst:39]: Added a version with a larger text size

**Comment by chrisjurich** [comment]
I have addressed the proposed comments. Thank you for the feedback!

Chris

**Review by shaoqx — APPROVED** [review]
Good job! Approved

**Pull request merged**